test/notifications/receive_test.dart

```diff
@@ -30,4 +30,12 @@ void main() {
         .length.equals(1);
     }, variant: const TargetPlatformVariant({TargetPlatform.android, TargetPlatform.iOS}));
   });
+
+  group('tokens', () {
+    test('APNs token registration using correct app bundle ID', () async {
+      await init();
+      check(await testBinding.getAppBundleId())
+        .equals('com.zulip.flutter.test');
+    }, );
+  });
```

Comment:
Hmm, this commit should be dropped as `getAppBundleId` doesn't exist anymore.

This is one reason that we should include the tests in the same commit that implements the feature tested.